front/app/components/IdeasMap/desktop/IdeaMapOverlay.tsx

```diff
@@ -173,43 +173,45 @@ const IdeaMapOverlay = memo<Props>(
             />
           )}
           {!isMobileOrSmaller && idea && (
-            <CSSTransition
-              classNames="animation"
-              in={true}
-              timeout={timeout}
-              mountOnEnter={true}
-              unmountOnExit={true}
-              enter={true}
-              exit={true}
-            >
-              <InnerOverlay
-                right={smallerThan1440px ? '-100px' : '-150px'}
-                tabIndex={-1}
-                // Ref to use to focus on the overlay after selecting an idea
-                ref={overlayRef}
+            <>
+              <CSSTransition
+                classNames="animation"
+                in={true}
+                timeout={timeout}
+                mountOnEnter={true}
+                unmountOnExit={true}
+                enter={true}
+                exit={true}
               >
-                <Box display="flex" mt="16px" ml="24px">
-                  <GoBackButton
-                    onClick={() => {
+                <InnerOverlay
+                  right={smallerThan1440px ? '-100px' : '-150px'}
+                  tabIndex={-1}
+                  // Ref to use to focus on the overlay after selecting an idea
+                  ref={overlayRef}
+                >
+                  {/* <Box display="flex" mt="16px" ml="24px">
+                    <GoBackButton
+                      onClick={() => {
+                        onSelectIdea(null);
+                      }}
+                    />
+                  </Box> */}
+                  <IdeaShowPageTopBar
```

Comment:
Without a phase, this component doesn't render, so we should take out that phase check in IdeaShowPageTopBar as I'm doing in this PR, or make it a required prop.